frontend/src/features/customer/components/cards/PaymentFailedCard.tsx

```diff
@@ -12,7 +12,7 @@ const PaymentFailedCard = () => {
         </div>
         <img src='/images/card-failed.svg' alt='결제 실패' />
       </div>
-      <Button variant='assistive' size='md'>
+      <Button variant='assistive' size='md' onClick={() => alert('준비중인 기능이에요')}>
```

Comment:
여기도 투두 추가해주시면 좋을 것 같습니다!
```suggestion
    //TODO: 추후 삭제
      <Button variant='assistive' size='md' onClick={() => alert('준비중인 기능이에요')}>
```